lib/rules/is-visible-on-screen-matches.js

```diff
@@ -0,0 +1,5 @@
+import { isVisibleOnScreen } from '../commons/dom';
+
+export default function isVisibleOnScreenMatches(node) {
+  return isVisibleOnScreen(node);
```

Comment:
Since we're at it, should we just swap this over to a virtual node?

Reply:
Good call, I'll do that